cgo/main_integration_test.go

```diff
@@ -2,15 +2,15 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package cgotest
+// +build integration
+
+package cgo
```

Comment:
Mhm, looking at this naming the file just `integration_test.go` would be better.